client/task-list/test/payments.js

```diff
@@ -69,7 +69,7 @@ describe( 'TaskList > Payments', () => {
 			label: 'Install',
 		};
 
-		it( 'shows "create account" when Jetpack and WCS are connected', async () => {
+		it.skip( 'shows "create account" when Jetpack and WCS are connected', async () => {
```

Comment:
I don't know the full context around the decoupling of the PayPal Plugin from WCS/Jetpack, but will we ever re-enable these tests?

Reply:
Likely not, I deleted this, as it will allow you to create a new account by default in the new paypal onboarding flow.
@pmcpinto as a double check, is this just for creating a Paypal account? or is this also doing something else that I am unaware off?

Reply:
Yes, I think so. AFAIR the WCS/JP was already removed from WCPayments and Stripe. Perhaps @thenbrent can confirm it

Reply:
Correct - the Stripe flow no longer requires WCS/JP plugins to use the OAuth connection flow, and WC Pay uses the Jetpack Connection Package (DNA) instead of requiring the Jetpack plugin so it doesn't require them either.